tar/multitape/multitape_fsck.c

```diff
@@ -165,7 +166,7 @@ phase1(uint64_t machinenum, STORAGE_D * SD, STORAGE_R * SR,
 		goto err0;
 
 	/* Allocate space for nfiles tapemetadata structures. */
-	if ((mdats = malloc(nfiles * sizeof(struct tapemetadata *))) == NULL)
+	if (IMALLOC(mdats,nfiles,struct tapemetadata *)) 
```

Comment:
Bad whitespace -- there should be spaces after the commas in a function call or macro, and there should not be any end-of-line whitespace.